spec/rack/test/cookie_object_spec.rb

```diff
@@ -6,8 +6,8 @@
 describe Rack::Test::Cookie do
   value = 'the cookie value'.freeze
   domain = 'www.example.org'.freeze
-  path = '/'.freeze
-  expires = 'Mon, 10 Aug 2015 14:40:57 0100'.freeze
+  path = '/foo'.freeze
+  expires = (Time.now + (24 * 60 * 60)).httpdate
```

Comment:
I've made this a date in the future so tests on `matches?` don't fail because of the previously expired cookie.